aiohttp/client.py

```diff
@@ -639,6 +620,27 @@ def ws_connect(
             max_msg_size: int=4*1024*1024
     ) -> ClientWebSocketResponse:
 
+        ws_timeout: ClientTimeout
+        if timeout is not sentinel:
+            if isinstance(timeout, ClientTimeout):
+                ws_timeout = timeout
+            else:
+                warnings.warn("parameter 'timeout' of type 'float' "
+                              "is deprecated, please use "
+                              "'timeout=ClientTimeout(sock_close=...)'",
+                              DeprecationWarning,
+                              stacklevel=2)
+                ws_timeout = ClientTimeout(sock_close=timeout)
+        else:
+            ws_timeout = DEFAULT_WS_CLIENT_TIMEOUT
```

Comment:
If I use `ws_connect(url)`, I think this would set `sock_read=10.0`. But in the [past](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client.py#L627), it should be `_receive_timeout=None`. So I think maybe it is not right?

Reply:
true. Thank you, I'll just change defaults in `DEFAULT_WS_CLIENT_TIMEOUT` for `receive_timeout` (i.e. `ws_receive`) to be `None`

Reply:
Or maybe instead we should set default receive timeout to `10` seconds? as well as closing socket timeout?

Reply:
Added this information to changelog notice

Reply:
I think we should keep the original one. And I’m using this as a feature. if not, I have a lot of work to do.

Reply:
`receive_timeout` should be `None` by default as we have it already.

Reply:
ok